tests/functional/aws-node-sdk/test/multipleBackend/put.js

```diff
@@ -372,14 +347,23 @@ describeSkipIfNotMultiple('MultipleBackend put object based on bucket location',
                 assert.equal(err, null, `Error creating bucket: ${err}`);
                 process.stdout.write('Putting object\n');
                 return s3.putObject(params, err => {
-                    assert.equal(err, null, 'Expected success, ' +
-                        `got error ${JSON.stringify(err)}`);
-                    s3.getObject({ Bucket: bucket, Key: key }, (err, res) => {
-                        assert.strictEqual(err, null, 'Expected success, ' +
-                        `got error ${JSON.stringify(err)}`);
-                        assert.strictEqual(res.ETag, `"${correctMD5}"`);
-                        done();
-                    });
+                    assert.equal(err, null,
+                        `Expected success, got error ${err}`);
+                    setTimeout(() => {
+                        s3.getObject({ Bucket: bucket, Key: key },
+                        (err, res) => {
+                            assert.strictEqual(err, null,
+                                `Expected success, got error ${err}`);
+                            assert.strictEqual(res.ETag, `"${correctMD5}"`);
+                            awsS3.getObject({ Bucket: awsBucket, Key: key },
+                            (err, res) => {
+                                assert.strictEqual(err, null,
+                                    `Expected success, got error ${err}`);
+                                assert.strictEqual(res.ETag, `"${correctMD5}"`);
+                                done();
+                            });
+                        });
+                    }, 10000);
```

Comment:
and this one? let's use a constant defined at the top of the file for the delay time.